packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -589,13 +589,38 @@ describeIf(process.platform !== 'win32' && !isMacOrWindowsCI)('MongoDB', () => {
     expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
     expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(`
 
-      // *** WARNING ***
-      // 
-      // The following fields had data stored in multiple types. The most common type was chosen. If loading data with a type that does not match the one in the data model, the client will crash. Please see the issue: https://github.com/prisma/prisma/issues/9654
-      // - Model "users", field: "numberOrString1", chosen data type: "Int32"
-      // - Type "UsersHobbies", field: "numberOrString2", chosen data type: "Int32"
-      // 
+                  // *** WARNING ***
```

Comment:
Why are we not getting the same output in this test as in the one below?

Reply:
this is using --print, below is using --force